models/public/deblurgan-v2/deblurgan-v2.md

```diff
@@ -0,0 +1,143 @@
+# deblurgan-v2
+
+## Use Case and High-Level Description
+DeblurGAN-v2 is a generative adversarial network (GAN) for single image motion deblurring.
+This model is based on a relativistic conditional GAN with a double-scale discriminator.
+For details about architecture of model, check out the [paper](https://arxiv.org/abs/1908.03826).
+Model used Mobilenet as backbone and was trained on GoPro, DVD, NFS datasets.
+For details about implementation of model,
+check out the [DeblurGAN-v2: Deblurring (Orders-of-Magnitude) Faster and Better](https://github.com/VITA-Group/DeblurGANv2) repository.
+
+## Specification
+
+GFLOPs and MParams are presented for Mobilenet backbone.
+
+| Metric            | Value           |
+|-------------------|-----------------|
+| Type              | ImageProcessing |
+| GFLOPs            | 80.8919         |
+| MParams           | 2.1083          |
+| Source framework  | PyTorch\*       |
+
+## Accuracy
+
+Model was tested on GoPro test dataset. Metrics are presented for Mobilenet backbone.
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| PSNR   | 28.25Db        | 28.25Db         |
+| SSIM   | 0.97           | 0.97            |
+
+## Input
+
+### Original Model
+
+Image, name - `blur_image`,  shape - `1,3,736,1312`, format is `B,C,H,W`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`. Mean values - [127.5,127.5,127.5], scale values - [127.5,127.5,127.5].
+
+### Converted Model
+
+Image, name - `blur_image`,  shape - `1,3,736,1312`, format is `B,C,H,W`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original Model
+
+Deblurred image, name - `deblur_image`,  shape - `1,3,736,1312`, output data format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
+
+### Converted Model
+
+Deblurred image, name - `deblur_image`,  shape - `1,3,736,1312`, output data format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
```

Comment:
why is it still RGB?
if normalization is not performed during MO conversion, it should be noted in this section

Reply:
Output in `RGB` format as in original model. For IR model using reverse_input_channels.